src/main/scala/io/sdkman/changelogs/Migrations.scala

```diff
@@ -47,4 +47,12 @@ class Migrations {
     insertVersions(CandidateVersion("scala", "2.12.5", Some("UNIVERSAL"), "https://downloads.lightbend.com/scala/2.12.5/scala-2.12.5.zip"))
     updateCandidateDefault("scala", "2.12.5")
   }
+  
+  @ChangeSet(order = "008",id ="008-add_openjdk_9_0_4",author="MaSven")
+  def migrate008(implicit db: MongoDatabase)=insertVersions(
+    CandidateVersion("java","9.0.4-openjdk",Some("MAC_OSX"),"https://download.java.net/java/GA/jdk9/9.0.4/binaries/openjdk-9.0.4_osx-x64_bin.tar.gz"),
```

Comment:
`CandidateVersion` is now renamed to `Version` for simplicity.

Reply:
Would you be able to move these into the `JavaMigrations` class please? `Migrations` is now deprecated.